aten/src/ATen/native/sparse/SparseCsrTensorMath.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <ATen/Tensor.h>
+#include <ATen/native/DispatchStub.h>
+/* #include <c10/util/Optional.h> */
+
+namespace at { namespace native {
+
+using linalg_solve_fn = void(*)(
+  const at::Tensor&, const at::Tensor&, at::Tensor&, int&
```

Comment:
It's better to always prefer const references
```suggestion
  const at::Tensor&, const at::Tensor&, const at::Tensor&, int&
```